File: demo1/src/main/java/org/mosquito/demo/dto/UserDto.java
package org.mosquito.demo.dto;

import org.mosquito.framework.core.model.BaseDto;

import lombok.Getter;
import lombok.Setter;


/**
 * 数据传输对象实体
 *
 * @author jcchen
 * @Date 2018-5-26
 * 
 */
@Getter
@Setter
public class UserDto extends BaseDto {

    private static final long serialVersionUID = 1L;

    /**
     * 
     * 
     */
    private String name;

    /**
     * 
     * 
     */
    private String password;

    /**
     * 
     * 
     */
    private String salt;

    /**
     * 
     * 
     */
    private String mobile;
	
}

File: demo1s/src/main/scala/org/mosquito/demo1s/dto/UserDto.scala
package org.mosquito.demo1s.dto

import org.mosquito.framework.core.model.BaseDto

import scala.beans.BeanProperty


/**
 * 数据传输对象实体
 *
 * @author jcchen
 * @Date 2018-5-28
 * 
 */
class UserDto extends BaseDto {


    /**
     * 
     * 
     */
    @BeanProperty var name: String = _

    /**
     * 
     * 
     */
    @BeanProperty var password: String = _

    /**
     * 
     * 
     */
    @BeanProperty var salt: String = _

    /**
     * 
     * 
     */
    @BeanProperty var mobile: String = _
	
}
